### ISSUE TITLE
Global symbol "$VERBOSE" requires explicit package name

### ISSUE THREAD

**Issue by eserte** [cxw42/unbelievable#3]
On one of my smokers the test suite fails:
```
#   Failed test 'use App::unbelievable;'
#   at t/00_compile.t line 4.
#     Tried to use 'App::unbelievable'.
#     Error:  Global symbol "$VERBOSE" requires explicit package name (did you forget to declare "my $VERBOSE"?) at /usr/home/cpansand/.cpan/build/2020021602/App-unbelievable-v0.0.4-4/blib/lib/App/unbelievable.pm line 321.
# Compilation failed in require at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.

#   Failed test 'use App::unbelievable::CLI;'
#   at t/00_compile.t line 4.
#     Tried to use 'App::unbelievable::CLI'.
#     Error:  Global symbol "$VERBOSE" requires explicit package name (did you forget to declare "my $VERBOSE"?) at /usr/home/cpansand/.cpan/build/2020021602/App-unbelievable-v0.0.4-4/blib/lib/App/unbelievable/CLI.pm line 86.
# Global symbol "$VERBOSE" requires explicit package name (did you forget to declare "my $VERBOSE"?) at /usr/home/cpansand/.cpan/build/2020021602/App-unbelievable-v0.0.4-4/blib/lib/App/unbelievable/CLI.pm line 86.
# BEGIN not safe after errors--compilation aborted at /usr/home/cpansand/.cpan/build/2020021602/App-unbelievable-v0.0.4-4/blib/lib/App/unbelievable/CLI.pm line 167.
# Compilation failed in require at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.
# Looks like you failed 2 tests of 2.
t/00_compile.t ..... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/2 subtests 
```

This seems to happen if vars::i is already installed, but too old --- 1.07 seems to be problematic, 1.10 and newer is OK.

**Comment by cxw42** [comment]
Thanks very much!  I saw that and wasn't sure why --- I appreciate your providing the fix!  I'll release an update later today (barring the unexpected).

**Comment by cxw42** [comment]
Should be fixed in v0.0.5 --- I'll leave this issue open until we have confirmation from the smokers.  Thanks again!

**Comment by eserte** [comment]
v0.0.5 looks good: http://matrix.cpantesters.org/?dist=App-unbelievable+v0.0.5
